### DISCUSSION TITLE
Deep Link URIs not accepted as redirect_uri under fief admin portal

### DISCUSSION THREAD

**Discussion by xingster** [fief-dev/fief#453] [Q&A]
Hi there, I found a small issue:
1. When adding "Client" -> "Redirect URI", adding a deep link custom url scheme (e.g., 'myapp://') returns "invalid url".

I updated the "redirect_uris" column in the fief database client table directly and it enabled the redirect flow to work with the mobile app's authsession flow.

Thanks!

**Comment by frankie567** [comment]
Indeed, our validation only accepts HTTP URL; but should also definitely accept custom app scheme. Adding this as a bug!

**Reply by frankie567** [reply]
Tracked in #454

**Comment by frankie567** [comment]
Hmm... After checking, it looks like it works 🤔 Can you share the value you tried that didn't work?

<img width="495" alt="Capture d’écran 2024-10-06 à 14 31 35" src="https://github.com/user-attachments/assets/5a00a81f-3627-49d7-a21e-f421e4b5b505">

**Reply by xingster** [reply]
You're right, the issue is only related to adding a custom scheme with the redirect being the root url (e.g., "myapp://").
<img width="449" alt="Screenshot 2024-10-06 at 10 31 22 PM" src="https://github.com/user-attachments/assets/b502b438-1e02-44a5-aff1-3882cdafdf14">

**Reply by frankie567** [reply]
Hmm, I see. In this case, I'm not sure we should accept it as a valid value. I'm not too familiar with mobile app development, but I guess you can always workaround that by adding an arbitrary path?

**Reply by xingster** [reply]
That sounds good, I can definitely work around this. Thank you!
